### PR TITLE
Df 1503 update content

### PR DESCRIPTION
**Pull request by jimmynotjim**
Update static content from Content Team

## Additions

- None

## Removals

- None

## Changes

- Updated `/the-bureau` with static content changes
- Updated `/budget` with static content changes
- Updated `/doing-business-with-us` with static content changes

## Testing

Locally navigate to:

- /the-bureau
- /budget
- /doing-business-with-us

## Review

- @jameshupp 
- @sebworks 
- @anselmbradford 

## Preview

the-bureau

budget

doing-business-with-us

## Notes

- There's some extra markup that could use some cleaning. Kind of outside the scope here but I'm going to add it to the Github issue tracker before merging this

### CODE REVIEW

**Comment by sebworks** [comment]
:+1:

File: budget/index.html [hunk 1]
```diff
@@ -67,42 +71,57 @@ <h2 class="h3">Our financial report</h2>
...
+            <div class="content-l content-l__main">
+                <section class="content-l_col content-l_col-1-2">
+                    <h1 class="header-slug">
+                        <span class="header-slug_inner">
+                            Doing business with the CFPB
+                        </span>
+                    </h1>
+                    <p class="short-desc">
+                        We work with vendors who can help us achieve our mission. We publish
+                        information information about both existing and future procurements.
```
Comment (anselmbradford) [inline budget/index.html]: Typo `information information`

File: doing-business-with-us/index.html [hunk 2]
```diff
@@ -18,38 +18,37 @@
...
-                    Register with System of Award Management
+                    Register with System of Award Management (SAM)
                 </h2>
                 <p class="short-desc">
-                    Processes for procurement and awards across the federal government
-                    are managed in the System of Award Management (SAM).
-                    Anyone interested in federal procurements or awards should
-                    first register with SAM.
+                    Processes for procurement and awards across the federal government are managed
+                    in the System of Award Management. Anyone interested in federal procurements or
```
Comment (anselmbradford) [inline doing-business-with-us/index.html]: Poss. small typo... I think `federal` would be capitalized?

File: the-bureau/index.html [hunk 3]
```diff
@@ -72,15 +86,17 @@ <h1 class="h3">Innovate</h1>
...
                     block__border-bottom">
         <h1 class="h2">Our core functions</h1>
-        <p>We work to give consumers the information they need to understand the terms of their agreements with financial companies. We are working to make regulations and guidance as clear and streamlined as possible so providers of consumer financial products and services can follow the rules on their own.</p>
-        
-        <div class="expandable 
-                    expandable__expanded 
-                    expandable__mobile-only 
-                    u-mb0 
+        <p>Congress created the CFPB to create a single point of accountability for enforcing
+        federal consumer financial laws and protecting consumers in the financial marketplace.
```
Comment (anselmbradford) [inline the-bureau/index.html]: Capitalize `federal`? The White House does it... http://www.whitehouse.gov/blog/2012/01/04/consumer-financial-protection-bureau-101-why-we-need-consumer-watchdog

File: doing-business-with-us/index.html (hunk 2 above)
Comment (Scotchester) [inline doing-business-with-us/index.html]: Nope. See [our written style guide (GHE link)](https://ghe.domain/flapjack/written-style-guide/blob/master/new-style-guide-structure.md#3-terms). Can't link directly there, but "federal" is in that Terms section.

File: the-bureau/index.html [hunk 4]
```diff
@@ -97,27 +113,30 @@ <h1 class="h2">Our core functions</h1>
             </button>
             <div class="expandable_content">
                 <section class="block__padded-top u-mb0 u-mt0">
-                    <h1 class="h4">Congress established the CFPB to protect consumers by carrying out federal consumer financial laws. Among other things, we:</h1>
+                    <h1 class="h4">Among other things, we:</h1>
                     <ul class="list list__branded bureau-functions_list u-mb0">
-                        <li>Write rules, supervise companies, and enforce federal consumer financial protection laws.</li>
-                        <li>Restrict unfair, deceptive, or abusive acts or practices.</li>
+                        <li>Write rules, supervise companies, and enforce federal consumer
```
Comment (anselmbradford) [inline the-bureau/index.html]: Capitalize `federal`?

File: doing-business-with-us/index.html (hunk 2 above)
Comment (anselmbradford) [inline doing-business-with-us/index.html]: :+1: Cool, thanks @Scotchester I'll note in my other comments.

File: the-bureau/index.html (hunk 3 above)
Comment (anselmbradford) [inline the-bureau/index.html]: Not needed https://github.com/cfpb/cfgov-refresh/pull/322#discussion_r25868311

File: the-bureau/index.html (hunk 4 above)
Comment (anselmbradford) [inline the-bureau/index.html]: Not needed https://github.com/cfpb/cfgov-refresh/pull/322#discussion_r25868311

**Comment by jimmynotjim** [comment]
Updated and ready for sign off. @jameshupp can you double check this as well?

**Comment by sebworks** [comment]
:+1:

**Pull request merged**